Cargo.toml

```diff
@@ -365,6 +365,7 @@ crucible-common = { git = "https://github.com/oxidecomputer/crucible", rev = "c9
 # NOTE: See above!
 csv = "1.3.0"
 curve25519-dalek = "4"
+daft = { path = "../daft/daft" }
```

Comment:
This needs to be updated to point to the main branch on github after https://github.com/oxidecomputer/daft/pull/16 is merged in.

Planner tests and clippy pass locally.